CovertInfiltration/Src/CovertInfiltration/Classes/X2Helper_Infiltration.uc

```diff
@@ -1160,4 +1160,29 @@ static protected function int SortMultiStepLerpSteps (MultiStepLerpStep A, Multi
 	if (A.X == B.X) return 0;
 
 	return A.X < B.X ? 1 : -1;
-}
\ No newline at end of file
+}
+
+static function HandlePostMissionPOI(XComGameState NewGameState, XComGameState_Activity ActivityState, XComGameState_MissionSite MissionState, bool bSuccess)
```

Comment:
No need to pass the mission state, just use `GetMissionStateFromActivity(ActivityState)`